package.json

```diff
@@ -12,11 +12,10 @@
     "build": "node-ts-scripts bundle src/index.ts --dts --format=commonjs --platform=node --outputDir=build"
   },
   "devDependencies": {
-    "node-ts-scripts": "^0.12.17"
+    "node-ts-scripts": "^0.27.2"
   },
   "dependencies": {
-    "@types/mongodb": "^3.5.27",
-    "mongodb": "^3.6.1"
+    "mongodb": "^4.9.0"
```

Comment:
Move `mongodb` to `peerDependencies`( version >= 4.9 or >=4.0 ) so we don't need to upgrading this anymore.

Reply:
[move `mongodb` to `peerDependencies`](https://github.com/heineiuo/mongo-client-daemon/pull/1/commits/32aa0d4130ba3046ab7f444e513e58d505a2de6f)